mifosng-android/src/main/java/com/mifos/api/local/MifosDatabase.java

```diff
@@ -12,5 +18,49 @@
     public static final String NAME = "Mifos";
 
     //Always Increase the Version Number
-    public static final int VERSION = 2;
+    public static final int VERSION = 3;
+
+    @Migration(version = MifosDatabase.VERSION, database = MifosDatabase.class)
+    public static class Migration1 extends AlterTableMigration<LoanAccount> {
```

Comment:
@mayank-kgp don't make add migration classes here. Add new package directory  `database` with name `migrationversion2` and add all migration classes there.